src/containers/shared/components/Currency.tsx

```diff
@@ -11,10 +11,16 @@ export interface Props {
   shortenIssuer?: boolean
 }
 
+/*
+  LP token identifier is the identifier for LP tokens. All issued LP tokens start with 03 so we can use this to
```

Comment:
There should be a TODO for this, since this isn't a robust way of checking for LPs

Reply:
there's a todo on the other comment you left, but I'll mark it here too so we get both when the change is made.